source/fundamentals/connection/tls.txt

```diff
@@ -272,9 +271,7 @@ provided by Netty.
 
    The driver tests with Netty version ``{+nettyVersion+}``
 
-To instruct the driver to use `io.netty.handler.ssl.SslContext <https://netty.io/4.1/api/io/netty/handler/ssl/SslContext.html>`__,
-use the `NettyStreamFactoryFactory.Builder.sslContext <{+api+}/apidocs/mongodb-driver-core/com/mongodb/connection/netty/NettyStreamFactoryFactory.Builder.html#sslContext(io.netty.handler.ssl.SslContext)>`__
-method. See the method documentation for details about the different `io.netty.handler.ssl.SslProvider <https://netty.io/4.1/api/io/netty/handler/ssl/SslProvider.html>`__
```

Comment:
This information is still needed, but it should point to the methods `com.mongodb.MongoClientSettings.Builder.transportSettings`, `com.mongodb.connection.NettyTransportSettings.Builder.sslContext`. It should also continue pointing to the class `io.netty.handler.ssl.SslProvider`.

The current replacement for this text:

> See the method documentation for details about the different `io.netty.handler.ssl.SslProvider <https://netty.io/4.1/api/io/netty/handler/ssl/SslProvider.html>`__

is weird, as it's unclear what method it refers to.